README.md

```diff
@@ -40,6 +37,36 @@ Once a process terminates, the streams will become closed but not unset.
 * `$stdout`
 * `$stderr`
 
+Each of these implement the underlying
+[`DuplexStreamInterface`](https://github.com/reactphp/stream#duplexstreaminterface)
+and you can use any of its events and methods as usual:
+
+```php
+$process->stdout->on('data', function ($chunk) {
```

Comment:
I don't care, but you need to change the line below to `echo $chunk;` :)
Sorry, for not being clear enough.